crates/runc/src/lib.rs

```diff
@@ -368,6 +368,22 @@ pub trait Spawner: Debug {
 impl Runc {
     async fn launch(&self, cmd: Command, combined_output: bool) -> Result<Response> {
```

Comment:
```
async fn launch(&self, cmd: Command, combined_output: bool) -> Result<Response> {
```
There are some problems here, the cmd variable needs to be mutable, but if on a non-Linux environment, cmd cannot be mutable. If you don't use a separate function name, the code looks ugly because you need to create a new mut cmd and use a #[cfg] block. Is there a better way?